pkg/web/templates/user_profile.html

```diff
@@ -57,7 +57,7 @@ <h1 class="font-bold text-xl">Change Password</h1>
           </div>
           <div class="mb-4">
             <label for="confirm-password" class="label-style pb-0.5">Confirm Password</label>
-            <input type="password" id="confirm-password" name="confirm-password" class="input-style" />
```

Comment:
This was removed so that `confirm-password` no longer appears as part of the request body. However, if we want to update the BE this can be reversed.

Reply:
that's fine - I didn't really mind it going with the request since the BE just ignores it, but it's best practice to minimize the amount of data transmitted, so thank you for removing it!

Reply:
I've applied the suggested change in this [commit](https://github.com/trisacrypto/envoy/pull/154/commits/a8c8ac5619acaf6b5904708376bc306d3bc5bf38). I'll go ahead and merge, but if you think it should be handled differently let me know and I can make an update.

I should note that I tried to return a bool instead of a string, but got an error preventing the condition from running on line 30 of the `user_list.html` partial.